lib/prompb/marshal.go

```diff
@@ -217,6 +217,16 @@ func (m *MetricMetadata) marshalToSizedBuffer(dst []byte) (int, error) {
 		i--
 		dst[i] = 0x8
 	}
+	if m.AccountID != 0 {
+		i = encodeVarint(dst, i, uint64(m.AccountID))
+		i--
+		dst[i] = 0x58
```

Comment:
@makasim , @Haleygo , it would be great replacing these "magic numbers" in the `lib/promb` with the actual bitset operations used for building these numbers (aka "tags") according to [the protobuf message encoding format](https://protobuf.dev/programming-guides/encoding/#structure). This will improve the readability and simplify the maintenance of this code in the future, without losing performance, since Go compiler performs constant calculations at compile time.

Reply:
Created an issue to track this: https://github.com/VictoriaMetrics/VictoriaMetrics/issues/9608. @Haleygo, will look when she has time.